aggregator/src/config.rs

```diff
@@ -162,16 +168,49 @@ pub struct JobDriverConfig {
     /// communicating with the helper. See [`reqwest::ClientBuilder::timeout`] for details.
     #[serde(default = "JobDriverConfig::default_http_request_timeout")]
     pub http_request_timeout_secs: u64,
+
+    /// The initial interval, in milliseconds, to wait before retrying a retryable HTTP request.
+    #[serde(default = "JobDriverConfig::default_retry_initial_interval_millis")]
+    pub retry_initial_interval_millis: u64,
+    /// The maximum interval, in milliseconds, to wait before retrying a retryable HTTP request.
+    #[serde(default = "JobDriverConfig::default_retry_max_interval_millis")]
+    pub retry_max_interval_millis: u64,
+    /// The maximum elapsed time, in milliseconds, to wait before giving up on retrying a retryable
+    /// HTTP request.
+    #[serde(default = "JobDriverConfig::default_retry_max_elapsed_time_millis")]
+    pub retry_max_elapsed_time_millis: u64,
 }
 
 impl JobDriverConfig {
+    pub fn retry_config(&self) -> ExponentialBackoff {
+        ExponentialBackoffBuilder::new()
+            .with_initial_interval(Duration::from_millis(self.retry_initial_interval_millis))
+            .with_max_interval(Duration::from_millis(self.retry_max_interval_millis))
+            .with_max_elapsed_time(Some(Duration::from_millis(
+                self.retry_max_elapsed_time_millis,
+            )))
+            .build()
+    }
+
     fn default_http_connection_timeout() -> u64 {
         10
     }
 
     fn default_http_request_timeout() -> u64 {
```

Comment:
```suggestion
    fn default_http_connection_timeout_secs() -> u64 {
        10
    }

    fn default_http_request_timeout_secs() -> u64 {
```

I don't love that some values are expressed in seconds and some are in milliseconds, but I'm also not sure it's worth changing. However, we should rename these existing functions to disambiguate.

Reply:
I'm also a little uncomfortable with different units, but I do think millis is appropriate for a retry interval for an HTTP request.

We could freely change it in this version, but since I'm going to backport this to `release/0.6` things would get tricky there.